rest/route/select.go

```diff
@@ -23,21 +25,21 @@ type selectTestsHandler struct {
 // this information directly to the test selector.
 type SelectTestsRequest struct {
 	// Project is the project identifier.
-	Project string `json:"project" bson:"project"`
+	Project string `json:"project"`
```

Comment:
nit: can unexport the request struct

Reply:
I believe swaggo needs response type to be exported so that it can generate the REST API docs. This struct is for both the request and response.